compiler/codegen/OMRCodeGenerator.cpp

```diff
@@ -1171,18 +1167,6 @@ TR::Register *OMR::CodeGenerator::allocateSinglePrecisionRegister(TR_RegisterKin
    return temp;
    }
 
-TR::Register *OMR::CodeGenerator::allocate64bitRegister()
```

Comment:
There is no longer a difference in the `allocateRegister` APIs. All registers are assumed to be 32-bit unless used in a 64-bit context. These are all deprecated. There is likely some code which used the `allocate64bitRegister` API but was changed to use the `allocateRegister` API since they are synonymous now. This results in a bunch of code which looks like this:

```
if (<some condition>) {
    reg = cg->allocateRegister();
} else {
    reg = cg->allocateRegister(); // Used to be allocate64bitRegister
}
```

Such code should be folded away if you see it in an area you're working with.